src/playsim/mapthinkers/a_quake.cpp

```diff
@@ -408,17 +408,17 @@ int DEarthquake::StaticGetQuakeIntensities(double ticFrac, AActor *victim, FQuak
 //
 //==========================================================================
 
-bool P_StartQuakeXYZ(FLevelLocals *Level, AActor *activator, int tid, int intensityX, int intensityY, int intensityZ, int duration,
+bool P_StartQuakeXYZ(FLevelLocals *Level, AActor *activator, int tid, double intensityX, double intensityY, double intensityZ, int duration,
 	int damrad, int tremrad, FSoundID quakesfx, int flags,
 	double waveSpeedX, double waveSpeedY, double waveSpeedZ, int falloff, int highpoint, 
 	double rollIntensity, double rollWave)
 {
 	AActor *center;
 	bool res = false;
 
-	if (intensityX)		intensityX = clamp(intensityX, 1, 9);
-	if (intensityY)		intensityY = clamp(intensityY, 1, 9);
-	if (intensityZ)		intensityZ = clamp(intensityZ, 1, 9);
+	intensityX = clamp<double>(intensityX, 0.0, 9.0);
+	intensityY = clamp<double>(intensityY, 0.0, 9.0);
```

Comment:
Is there a particular reason this is 0.0 to 9.0 instead of 1-9?

Reply:
It allows for setting sub 1.0 intensities, like a really small earthquake with an intensity of 0.5.